From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Tonymac32 <tonymckahan@gmail.com>
Date: Mon, 18 Sep 2017 15:04:14 +0000
Subject: [ARCHEOLOGY] Add files via upload

> X-Git-Archeology: - Revision 3d96c26b0f4d6f94f1be3b42c1599f74a0f93465: https://github.com/armbian/build/commit/3d96c26b0f4d6f94f1be3b42c1599f74a0f93465
> X-Git-Archeology:   Date: Mon, 18 Sep 2017 15:04:14 +0000
> X-Git-Archeology:   From: Tonymac32 <tonymckahan@gmail.com>
> X-Git-Archeology:   Subject: Add files via upload
> X-Git-Archeology:
> X-Git-Archeology: - Revision 97d06e86c19d3d7d73c0a978fbad7d022b915922: https://github.com/armbian/build/commit/97d06e86c19d3d7d73c0a978fbad7d022b915922
> X-Git-Archeology:   Date: Mon, 18 Sep 2017 15:05:37 +0000
> X-Git-Archeology:   From: Tonymac32 <tonymckahan@gmail.com>
> X-Git-Archeology:   Subject: RK3288 get cpu to thermal_zone0
> X-Git-Archeology:
> X-Git-Archeology: - Revision 7ee8ff8e3bd040e4cc02596327b01c5f89dc1027: https://github.com/armbian/build/commit/7ee8ff8e3bd040e4cc02596327b01c5f89dc1027
> X-Git-Archeology:   Date: Mon, 18 Sep 2017 18:09:24 +0000
> X-Git-Archeology:   From: Tonymac32 <tonymckahan@gmail.com>
> X-Git-Archeology:   Subject: rk3288 dtsi thermal zone adjustment
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0d2da12129218170ac244cc8d6fc1b94e6f49089: https://github.com/armbian/build/commit/0d2da12129218170ac244cc8d6fc1b94e6f49089
> X-Git-Archeology:   Date: Sun, 17 Dec 2017 20:33:45 -0500
> X-Git-Archeology:   From: tonymac32 <tonymckahan@gmail.com>
> X-Git-Archeology:   Subject: Re-fix RK3288 Next Thermal Zones
> X-Git-Archeology:
> X-Git-Archeology: - Revision f831a2a014d9305e272fc7541fa7d85f32c7dbab: https://github.com/armbian/build/commit/f831a2a014d9305e272fc7541fa7d85f32c7dbab
> X-Git-Archeology:   Date: Mon, 19 Feb 2018 00:30:58 -0500
> X-Git-Archeology:   From: tonymac32 <tonymckahan@gmail.com>
> X-Git-Archeology:   Subject: Rockchip Dev fixup.  Boots.
> X-Git-Archeology:
> X-Git-Archeology: - Revision 150ac0c2afa147d9e3b036c8ecd8238fe5648cf3: https://github.com/armbian/build/commit/150ac0c2afa147d9e3b036c8ecd8238fe5648cf3
> X-Git-Archeology:   Date: Tue, 19 Nov 2019 23:25:39 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Remove K<4, change branches, new features (#1586)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 0cdffb29b07305209efb12cf3b5ac6032d3a1153: https://github.com/armbian/build/commit/0cdffb29b07305209efb12cf3b5ac6032d3a1153
> X-Git-Archeology:   Date: Wed, 24 Mar 2021 19:01:53 +0100
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Renaming DEV branch to EDGE (#2704)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 3b3d85e25c2ecde30df7b5274fc6f1b9c0299ea2: https://github.com/armbian/build/commit/3b3d85e25c2ecde30df7b5274fc6f1b9c0299ea2
> X-Git-Archeology:   Date: Sat, 22 May 2021 17:08:44 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Upgrade EDGE to 5.12.y (#2825)
> X-Git-Archeology:
> X-Git-Archeology: - Revision e7377248b3cae186e24e2be781cd3365b43246f0: https://github.com/armbian/build/commit/e7377248b3cae186e24e2be781cd3365b43246f0
> X-Git-Archeology:   Date: Thu, 22 Jul 2021 00:15:54 +0200
> X-Git-Archeology:   From: Igor Pecovnik <igorpecovnik@users.noreply.github.com>
> X-Git-Archeology:   Subject: Second part of EDGE bumping to 5.13.y (#3045)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 06d5054cce5d269b3fe8a0e23918c0c31a91140c: https://github.com/armbian/build/commit/06d5054cce5d269b3fe8a0e23918c0c31a91140c
> X-Git-Archeology:   Date: Sun, 24 Sep 2023 19:22:37 +0200
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip: bump edge kernel to 6.5
> X-Git-Archeology:
> X-Git-Archeology: - Revision f08dcd48677d2a34f349bf571c979cd422bffcc3: https://github.com/armbian/build/commit/f08dcd48677d2a34f349bf571c979cd422bffcc3
> X-Git-Archeology:   Date: Tue, 31 Oct 2023 08:13:23 +0100
> X-Git-Archeology:   From: Paolo <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip,rk322x: bump edge kernel to 6.6 (#5875)
> X-Git-Archeology:
> X-Git-Archeology: - Revision 562d96128ba6a511a8a06c0f4d29946ab80b8969: https://github.com/armbian/build/commit/562d96128ba6a511a8a06c0f4d29946ab80b8969
> X-Git-Archeology:   Date: Tue, 26 Dec 2023 16:45:30 +0100
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: consolidate rk322x and rockchip 32 bit families
> X-Git-Archeology:
> X-Git-Archeology: - Revision 47d2e8287e34fed3e47f37ab076d0f34ed0ac399: https://github.com/armbian/build/commit/47d2e8287e34fed3e47f37ab076d0f34ed0ac399
> X-Git-Archeology:   Date: Mon, 25 Mar 2024 19:38:38 +0100
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip: bump edge kernel to 6.8
> X-Git-Archeology:
> X-Git-Archeology: - Revision 724573bf7a21e61b0b626f835031a4c3206bb8ba: https://github.com/armbian/build/commit/724573bf7a21e61b0b626f835031a4c3206bb8ba
> X-Git-Archeology:   Date: Wed, 05 Jun 2024 22:18:51 +0200
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: bump rockchip family edge kernel to 6.9
> X-Git-Archeology:
> X-Git-Archeology: - Revision 7da7bbf61cb776a054219e35926d391dad9a67a7: https://github.com/armbian/build/commit/7da7bbf61cb776a054219e35926d391dad9a67a7
> X-Git-Archeology:   Date: Mon, 22 Jul 2024 19:18:14 +0200
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip: bump edge kernel to 6.10
> X-Git-Archeology:
> X-Git-Archeology: - Revision 94ec783de0dad381b3e2e71d646d8428af4d5051: https://github.com/armbian/build/commit/94ec783de0dad381b3e2e71d646d8428af4d5051
> X-Git-Archeology:   Date: Wed, 18 Sep 2024 14:03:19 +0200
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: bump rockchip 32 bit edge kernel to 6.11
> X-Git-Archeology:
> X-Git-Archeology: - Revision c90a0f7890bddc8e755847fc8227e15828950251: https://github.com/armbian/build/commit/c90a0f7890bddc8e755847fc8227e15828950251
> X-Git-Archeology:   Date: Sat, 30 Nov 2024 13:07:31 +0100
> X-Git-Archeology:   From: Paolo Sabatino <paolo.sabatino@gmail.com>
> X-Git-Archeology:   Subject: rockchip: bump edge kernel to 6.12
> X-Git-Archeology:
---
 arch/arm/boot/dts/rockchip/rk3288.dtsi | 14 +++++-----
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/arch/arm/boot/dts/rockchip/rk3288.dtsi b/arch/arm/boot/dts/rockchip/rk3288.dtsi
index 111111111111..222222222222 100644
--- a/arch/arm/boot/dts/rockchip/rk3288.dtsi
+++ b/arch/arm/boot/dts/rockchip/rk3288.dtsi
@@ -457,13 +457,6 @@ dmac_peri: dma-controller@ff250000 {
 	};
 
 	thermal-zones {
-		reserve_thermal: reserve-thermal {
-			polling-delay-passive = <1000>; /* milliseconds */
-			polling-delay = <5000>; /* milliseconds */
-
-			thermal-sensors = <&tsadc 0>;
-		};
-
 		cpu_thermal: cpu-thermal {
 			polling-delay-passive = <100>; /* milliseconds */
 			polling-delay = <5000>; /* milliseconds */
@@ -535,6 +528,13 @@ map0 {
 				};
 			};
 		};
+
+		reserve_thermal: reserve-thermal {
+			polling-delay-passive = <1000>; /* milliseconds */
+			polling-delay = <5000>; /* milliseconds */
+
+			thermal-sensors = <&tsadc 0>;
+		};
 	};
 
 	tsadc: tsadc@ff280000 {
-- 
Armbian

